crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -42,24 +48,53 @@ pub struct StandardMaterial {
     /// it just adds a value to the color seen on screen.
     pub emissive: Color,
 
+    /// The [`emissive`] as a texture.
+    ///
+    /// This color is multiplied by [`emissive`] to get the final emitted color.
+    /// Meaning that you should set [`emissive`] to [`Color::WHITE`]
+    /// if you want to use an emissive texture.
+    ///
+    /// [`emissive`]: StandardMaterial::emissive
     #[texture(3)]
     #[sampler(4)]
     pub emissive_texture: Option<Handle<Image>>,
-    /// Linear perceptual roughness, clamped to [0.089, 1.0] in the shader
-    /// Defaults to minimum of 0.089
+
+    /// Linear perceptual roughness, clamped to `[0.089, 1.0]` in the shader.
+    ///
+    /// Defaults to minimum of `0.089`.
+    ///
+    /// Low values result in a "glossy" material with specular highlights,
+    /// while values close to `1` result in rough materials.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `roughness * roughness_texture_value`
+    /// color as `roughness * roughness_texture_value`.
     pub perceptual_roughness: f32,
-    /// From [0.0, 1.0], dielectric to pure metallic
+
+    /// How "metallic" the material appears, within `[0.0, 1.0]`,
+    /// going from dielectric to pure metallic.
+    ///
+    /// Defaults to `0.01`.
+    ///
+    /// The closer to `1` the value, the more the material will
+    /// reflect light like a metal such as steel or gold.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `metallic * metallic_texture_value`
+    /// color as `metallic * metallic_texture_value`.
     pub metallic: f32,
 
+    /// A texture representing both [`metallic`] and [`roughness`].
+    ///
+    /// The blue channel is the [`metallic`] and green is [`roughness`].
+    ///
+    /// [`metallic`]: StandardMaterial::metallic
+    /// [`roughness`]: StandardMaterial::perceptual_roughness
     #[texture(5)]
     #[sampler(6)]
     pub metallic_roughness_texture: Option<Handle<Image>>,
-    /// Specular intensity for non-metals on a linear scale of [0.0, 1.0]
-    /// defaults to 0.5 which is mapped to 4% reflectance in the shader
+
+    /// Specular intensity for non-metals on a linear scale of `[0.0, 1.0]`.
+    ///
+    /// Defaults to `0.5` which is mapped to 4% reflectance in the shader
     pub reflectance: f32,
```

Comment:
If the doc is correct then that should be named `specular_intensity` not `reflectance`. Is there a bug to track that rename?

Reply:
The naming is proper to PBR. `specular_intensity` is what you would call this on a classic phong shader, but by convention, PBR uses the term "reflectance" inherited from physics. Bevy re-uses the Google Filament naming scheme seen at <https://google.github.io/filament/Material%20Properties.pdf>.

I agree though. I thought of adding `#[doc(alias = "specular_intensity")]`, but I was worried this would cause more eyebrow raising, so I keept with the status quo.

Reply:
My point is more simple: if the parameter "maps to" reflectance, then that means it's not itself reflectance, otherwise it would map to itself (1:1) which the docs clearly says is not the case (0.5:4%). So there's something wrong somewhere.

Reply:
Sure, I'll open a discussion. For now, I added the doc alias.

Reply:
I still don't understand how to use that parameter. If I want to achieve 4% reflectance, should I set `StandardMaterial::reflectance = 0.04` (and it will map to specular intensity of `0.5`), or the opposite, I should set `StandardMaterial::reflectance = 0.5` because it's actually (despite the name) the specular intensity and will be remapped internally to 4% reflectance? I simply don't understand that documentation.

Reply:
I agree with what you are saying here. But there are caveats to consider with regard to what bevy users are familiar with. I opened [a discussion](https://github.com/bevyengine/bevy/discussions/5936) so that it the discussion is not buried in a PR thread. I'll add some verbiage to explicit the distinction, though I'll keep it named `reflectance` until the question is resolved in the linked discussion thread.